packages/travix-ui-kit/components/dropDown/dropDown.scss

```diff
@@ -8,6 +8,12 @@ $select-option-color: var(--tx-dropdown-option-color);
 
 $select-arrow-color: var(--tx-dropdown-arrow-color);
 
+.Select.ui-dropdown.is-open {
```

Comment:
Why don't you nest this rule `.ui-dropdown` as other rules have been?

Reply:
@vitormargis I had to put more specific rule, as otherwise I can't override the styles from react select :( (the rule .Select.is-open)

Reply:
This is equivalent to:

```scss
.ui-dropdown {
  &.Select.is-open {...}
}
```

So it can be nested under `.ui-dropdown` 🙂

Reply:
@vitormargis @rbardini this way doesn't work, doesn't override react-select